config/e2e/namespace_operator.yaml

```diff
@@ -167,15 +165,18 @@ roleRef:
   name: elastic-operator
 subjects:
 - kind: ServiceAccount
-  name: {{ .Name }}
-  namespace: {{ .Namespace }}
+  name: {{ .NamespaceOperator.Name }}
+  namespace: {{ .NamespaceOperator.Namespace }}
+
+{{- $nsOperator := .NamespaceOperator -}}
```

Comment:
not exactly sure I understand why we sometimes use `$nsOperator` vs. `.NamespaceOperator`?

Reply:
It's because the following `range` block shadows the original object and it is no longer possible to use the dot notation to access attributes of the parent object without saving it to a variable first.